packages/amplify-cli/src/extensions/amplify-helpers/get-project-details.js

```diff
@@ -12,11 +13,7 @@ function getProjectDetails() {
     amplifyMeta = readJsonFile(amplifyMetaFilePath);
   }
 
-  let localEnvInfo = {};
-  const envFilePath = pathManager.getLocalEnvFilePath();
-  if (fs.existsSync(envFilePath)) {
-    localEnvInfo = readJsonFile(envFilePath);
-  }
+  const localEnvInfo = getEnvInfo();
```

Comment:
What if the file doesn't exist? Will it throw an error right? 
I think in the previous logic we wern't throwing an error when file doesn't exist and just checking for its existence.

Reply:
I visited all the places where we were reading this files and we're consuming straight ahead what was returned, without checking it, that's why I added the throw to ```getEnvInfo``` itself. LMK if you find a place where it is not the case, I could missed that, but tried to be thorough.